qcsrc/localized_text.qc

```diff
@@ -65,17 +65,17 @@
 
 #ifdef __LIBREQUAKE__
 
-#define LOC_CLIENT_JOIN             " joined the game"
-#define LOC_CLIENT_DISCON           " exited the level"
-#define LOC_CLIENT_DISCON_DM_A      " left the game with "
-#define LOC_CLIENT_DISCON_DM_B      " frags"
+#define LOC_CLIENT_JOIN				" joined the server"
```

Comment:
Was changing the indentation here your doing or the editors?

Reply:
Sorry. I meant to fix the screwy indentation in this file when I was done with anything, but I forgot. I'll do that now.